statistics-production/api-data-standards.qmd

```diff
@@ -0,0 +1,191 @@
+
+::: {.table-responsive}
+
+|attainment_metric   | count_pupils | percent_pupils |
+|--------------------|--------------|----------------|
+| Grades 9-5         | 30           | 60             |          
+| Grades 9-4         | 40           | 80             |         
+| Grades 9-1         | 50           | 100            |                  
+
+: Attainment grade rates in tidy format
+
+:::
+
+::: {.table-responsive}
+
+| sex    | attainment_metric | score_average  |
+|--------|-------------------|----------------|
+| Female | Progress 8        | 0.21           |          
+| Male   | Progress 8        | 0.20           |         
+| Total  | Progress 8        | 0.21           |                  
+| Female | Attainment 8      | 0.08           |          
+| Male   | Attainment 8      | 0.09           |         
+| Total  | Attainment 8      | 0.08           |                  
+
+: Attainment scores in tidy format
+
+:::
+
+### Example 2 - Pupil counts, percents and characteristics
+
+#### Example of bad practice
+
+::: {.table-responsive}
+
+| count_schools | count_pupils_male | count_pupils_female | count_pupils_total | percent_pupils_male | percent_pupils_female | percent_pupils_total |
+|---------------|-------------------|---------------------|--------------------|---------------------|-----------------------|----------------------|
+| 2             | 120               | 130                 | 250               | 48                  | 52                    | 100                  |
+
+: Pupil counts and percentages in non-tidy format
+
+:::
+
+#### Example of good practice
+
+::: {.table-responsive}
+
+| sex                | count_schools | count_pupils | percent_pupils |
+|--------------------|---------------|--------------|----------------|
+| Male               | 2             | 30           | 60             |          
+| Female             | 2             | 40           | 80             |         
+| Total              | 2             | 50           | 100            |                  
+
+: Pupil counts and percentages in tidy format
+
+:::
+
+###
+
+All data uploaded to EES should be in a tidy data structure form, but this is more strictly regulated for data intended for use with the API. More information on building tidy data structures can be found in the [tidy data structure section](../satistics-production/ud.html#tidy-data-structure).
+
+## Standardised filter col_names and items
+
+The explore education and statistics platforms team alongside the data harmonisation champions group and publication teams are developing a series of [standardised filter-sets](../statistics-production/ud.html#common-harmonised-variables) that teams are required to adhere to when creating data for the API. These are being built iteratively as more data is put forward for the API, so if the current standards don't cater to your data set, you can contribute to building the harmonised standards for others to follow.
```

Comment:
Is filter-sets a word? Reads a little odd (though don't have better suggestions)

Reply:
I've tried "standardised filter specifications". Open to other suggestions and also happy to switch back to filter sets (guess was maybe just using sets from the Maths-type context point of view).

Reply:
How about just 'standardised filters' - it's vague but covers everything (cols and items) and people will know what it means?

And then in the sentence following it would be `...that teams are required to use when creating data for the API...` rather than `adhere to`

Reply:
Done